packages/shorebird_cli/lib/src/shorebird_flutter.dart

```diff
@@ -121,6 +121,32 @@ class ShorebirdFlutter {
     return match?.group(1);
   }
 
+  /// Executes `flutter config --list` and returns the output as a map.
+  Map<String, dynamic> getConfig() {
+    final args = ['config', '--list'];
+    final result = process.runSync(executable, args);
+    if (result.exitCode != ExitCode.success.code) {
+      throw ProcessException(
+        executable,
+        args,
+        '${result.stderr}',
+        result.exitCode,
+      );
+    }
+    final output = '${result.stdout}';
+    final config = <String, dynamic>{};
+    final lines = LineSplitter.split(output).toList();
+    for (final line in lines.skip(1)) {
+      final parts = line.split(':');
+      if (parts.length == 2) {
```

Comment:
This will fail if the key has a `:` in it.  I doubt it matters, but you probably want to split on first-colon and use the remainer as the value.